ci/ray_ci/BUILD.bazel

```diff
@@ -37,7 +37,6 @@ py_binary(
 py_binary(
     name = "build_in_docker",
     srcs = ["build_in_docker.py"],
-    exec_compatible_with = ["//:hermetic_python"],
```

Comment:
could you add this back? why remove?

Reply:
the hermetic python doesn't work on windows